assessment/showtest.php

```diff
@@ -2087,14 +2089,14 @@
 			}
 
 			$query = "UPDATE imas_livepoll_status SET curquestion='$qn',curstate='$newstate' WHERE assessmentid='$aid'";
-			mysql_query($query) or die("Query failed : " . mysql_error());
+			mysql_query($query) or die(_("Query failed") . " : " . mysql_error());
 
 			$r = file_get_contents('https://'.$CFG['GEN']['livepollserver'].':3000/stopq?aid='.$aid.'&qn='.$qn.'&newstate='.$newstate.'&now='.$now.'&sig='.$livepollsig);
 
 			if ($r=='success') {
 				echo '{success: true}';
 			} else {
-				echo '{error: "'.$r.'"}';
+				echo '{error: "$r"}';

```

Comment:
Ditto. Since the $r is inside single quotes, this will prevent the variable interpolation from happening.